pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.isort]
+profile = "black"
+sections = [
+  "FUTURE",
+  "STDLIB",
+  "THIRDPARTY",
+  "ARCHSPEC", "LLNL", "FIRSTPARTY",
+  "LOCALFOLDER",
+]
+known_first_party = "spack"
+known_archspec = "archspec"
+known_llnl = "llnl"
+src_paths = "lib"
+force_sort_within_sections = false
+line_length = 79
```

Comment:
Shouldn't this be 88 (the default for `profile = 'black'`)?

Reply:
I guess if we're switching to black anyway we should go ahead and do it.